src/lib/DataConnectStore.js

```diff
@@ -59,6 +59,22 @@ export default class DataConnectStore {
     return useCase.connectors.map(c1 => this.find(c2 => c1.slug === c2.slug))
   }
 
+  // Fetch all accounts and updates their matching connectors
+  fetchAllAccounts (index) {
+    if (!index && this.accountsIndex) index = this.accountsIndex
```

Comment:
Do the same code in every store of future applications ... instead of setting it in one place one for once. Remember that `lib/accounts` and `lib/konnectors` are planned to be part of CozyClient.
What the point of keeping an index into the store ? For what feature could it be useful ?